packages/core/src/modules/connections/DidExchangeProtocol.ts

```diff
@@ -479,7 +479,7 @@ export class DidExchangeProtocol {
     const payload = JsonEncoder.toBuffer(json)
     const { isValid, signerVerkeys } = await this.jwsService.verifyJws({ jws, payload })
 
-    const didDocument = JsonTransformer.fromJSON(json, DidDocument)
+    const didDocument = JsonTransformer.fromJSON(json, DidDocument, { validate: false })
```

Comment:
What kind of errors do we get here, as this isn't a test file we should try to fix the issues instead of disabling validation